myob-api.gemspec

```diff
@@ -21,5 +21,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "oauth2", "> 0.8"
 
   spec.add_development_dependency "bundler", "~> 1.3"
+  spec.add_development_dependency "minitest", "~> 5.4"
```

Comment:
There's an issue with CircleCI regarding minitest versions and (at the minimum) ruby 2.2. Being explicit about which version should be used clears this up.